src/client/cli/cmd/exec.cpp

```diff
@@ -100,6 +118,10 @@ mp::ParseCode cmd::Exec::parse_args(mp::ArgParser* parser)
     parser->addPositionalArgument("name", "Name of instance to execute the command on", "<name>");
     parser->addPositionalArgument("command", "Command to execute on the instance", "[--] <command>");
 
+    QCommandLineOption workDirOption({"w", work_dir_option_name}, "Change to <dir> before execution", "dir");
```

Comment:
Maybe "d" would be a better short version?

Reply:
Right, `-d` sounds better. Will change it.